### PR TITLE
Chart data

### PR DESCRIPTION
**Pull request by czhou578**
I tried implementing the backend for NewCharts. I am getting a response on the frontend, although the specific fields are all 0's, even after I leave reviews on the certain vendor. Feedback will be appreciated.

### CODE REVIEW

File: backend/api.go [hunk 1]
```diff
@@ -99,6 +99,8 @@ func (a *API) AddRoutes(router *gin.Engine) {
 	router.GET("/discounts", a.Discounts)
 	router.GET("/discounts/:id", GetToken, a.Discount)
 	router.DELETE("/discounts/:id", GetToken, a.DiscountDelete)
+
+	router.GET("/newchart", a.NewChart)
```
Comment (MakotoE) [inline backend/api.go:103]: Since we have multiple charts, we should make this a sub-directory like `/charts/chart2`

**Review by MakotoE — CHANGES_REQUESTED** [review]

File: backend/api.go (hunk 1 above)
Comment (czhou578) [reply backend/api.go:103]: If I do like `/charts/starsumchart`, will that work?

Comment (MakotoE) [reply backend/api.go:103]: That is fine.

Comment (czhou578) [reply backend/api.go:103]: Is everything else ok? Do you see any additional problems?

Comment (czhou578) [reply backend/api.go:103]: image
Not sure what this error is: it's causing the reset database action to fail.

Comment (MakotoE) [reply backend/api.go:103]: This was actually fixed in a new commit to main. If you merge main into your branch, this should be fixed.

Comment (czhou578) [reply backend/api.go]: Ok. I fixed it and all checks have passed.

**Pull request merged**